tests/foreman/ui/test_dashboard.py

```diff
@@ -180,13 +182,22 @@ def test_positive_task_status(session):
         session.dashboard.action({
             'TaskStatus': {'state': 'running', 'result': 'pending'}
         })
-        tasks = session.task.read_all()
-        assert tasks['searchbox'] == 'state=running&result=pending'
+        searchbox = session.task.read_widget('searchbox')
+        assert searchbox == 'state=running&result=pending'
+        session.task.set_chart_filter('RunningChart')
+        row_count = session.task.table_row_count()
+        chart = session.task.read_widget('RunningChart')
+        assert row_count == int(chart['total']['Total'])
         session.dashboard.action({
             'TaskStatus': {'state': 'stopped', 'result': 'warning'}
         })
         tasks = session.task.read_all()
         assert tasks['searchbox'] == 'state=stopped&result=warning'
+        session.task.set_chart_filter(
+            'StoppedChart', {'row': 1, 'focus': 'Total'})
+        row_count = session.task.table_row_count()
+        chart = session.task.read_widget('StoppedChart')
+        assert row_count == int(chart['table'][1]['Total'])
```

Comment:
Do you think we should also validate the count present on dashboard?

Reply:
This could be a bit problem when you run tests in more threads. By the time test read whole dashboard and navigate to 'Tasks' this count could not be same.